docs/en/base/typography.md

```diff
@@ -63,6 +63,15 @@ modified to look like a particular heading size.
     View example of the heading pattern
 </a>
 
+### Sub-headings
+
+Sub-headings are used to visually convey importance beneath a heading or as a title when inserted as a divider between sections.
```

Comment:
“a title when inserted as a divider between sections” sounds very much like just a subsection heading `<h2>`.

I think what we mean by subheading, here, is “a line of text that expands on the meaning of the heading immediately before it, rather than being a standalone heading for the text after it”.